airflow/dag_processing/bundles/git.py

```diff
@@ -236,6 +238,8 @@ def view_url(self, version: str | None = None) -> str | None:
             return None
         if url.startswith("git@"):
             url = self._convert_git_ssh_url_to_https(url)
+        if url.endswith(".git"):
+            url = url[:-4]
```

Comment:
nit. How about using `len(".git")` instead of the hardcoded value 4 to make it more meaningful?
```suggestion
            url = url[:-len(".git")]
```

Reply:
imo. I don't think len is more meaningful. `.git` format won't change and might even seem more complex.

Reply:
Generally I agree @josix - but in this case being right under the `endswith` condition, the int is easier to read imo.